### PR TITLE
Added most recent setup for lstm containing label-delay.

### PR DESCRIPTION
**Pull request by pegahgh**

### CODE REVIEW

**Comment by danpovey** [comment]
OK, merging since this is urgent. 
 I notice you are using iVectors.  It's not really my aim to make use of iVectors in this setup if we can avoid it, but I'll check in your changes.

File: egs/ami/s5/local/nnet3/run_lstm.sh [hunk 1]
```diff
@@ -10,28 +10,27 @@ set -e
 stage=0
 train_stage=-10
 has_fisher=true
-mic=ihm
-use_sat_alignments=true
+mic=sdm1
```
Comment (vijayaditya) [inline egs/ami/s5/local/nnet3/run_lstm.sh]: this is a common script used for sdm*, ihm and mdm8. Please don't change the default mic. The default recipe for ihm uses sat alignments so don't change use_sat_alignments.

File: egs/ami/s5/local/nnet3/run_lstm.sh [hunk 2]
```diff
@@ -10,28 +10,27 @@ set -e
...
 affix=
 speed_perturb=true
-common_egs_dir=
-
-# lstm options
-splice_indexes="-2,-1,0,1,2"
-num_lstm_layers=1
+splice_indexes="-2,-1,0,1,2 0 0"
+label_delay=5
+num_lstm_layers=3
```
Comment (vijayaditya) [inline egs/ami/s5/local/nnet3/run_lstm.sh]: please don't modify the default values for splice_indexes and num_lstm_layers.

File: egs/ami/s5/local/nnet3/run_lstm.sh [hunk 3]
```diff
@@ -79,28 +81,33 @@ if [ $stage -le 7 ]; then
...
+  steps/nnet3/lstm/train_delay.sh --stage $train_stage \
+    --label-delay $label_delay \
+    --num-epochs $num_epochs --num-jobs-initial 2 --num-jobs-final 12 \
     --num-chunk-per-minibatch $num_chunk_per_minibatch \
     --samples-per-iter $samples_per_iter \
     --splice-indexes "$splice_indexes" \
     --feat-type raw \
     --online-ivector-dir exp/$mic/nnet3/ivectors_${train_set}_hires \
     --cmvn-opts "--norm-means=false --norm-vars=false" \
+    --io-opts "-tc 12" \
```
Comment (vijayaditya) [inline egs/ami/s5/local/nnet3/run_lstm.sh]: io-opts is not longer necessary for the new get_egs.sh script, you can remove that.

Comment (danpovey) [inline egs/ami/s5/local/nnet3/run_lstm.sh]: You can make these changes a separate pull request.
But don't you think a 3-layer LSTM is a more reasonable default?
Dan
